andela/Titan-Devs
# #163756376 User Can Login

#### What does this PR do?
Creates login functionality
#### Description of Task to be completed?
* Create `POST : /api/v1/users/login` route
* Create tests for `POST : /api/v1/users/login` route
* Create validation of login credentials. 
* Create login controller. 
#### How should this be manually tested?
* Fetch the branch `ft-user-can-login-163756376`
* Checkout the branch `ft-user-can-login-163756376`
* Setup your database. 
* Run tests `yarn run test`
* Start the server `yarn run server`
* Create a user `POST: /api/v1/users` with `email, password, and username`
* Consume the endpoint `POST: /api/v1/users/login ` with `email and password` using Postman client. 
* You should see a successfull login message. 
#### What are the relevant pivotal tracker stories?
163756376
